docs/source/query-plans.mdx

```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
     * Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 * As direct output from the `@apollo/gateway` library (see below)
 
+### Outputting query plans with headers
+In the latest versions of Apollo Router and Gateway you can pass the following headers to return the query plans in the response extensions:
+
+* Sending `Apollo-Query-Plan-Experimental` will return the query plan in the response extensions
+* Sending `Apollo-Query-Plan-Experimental-Format` with one of the supported options will change the format:
```

Comment:
Is this one or the other, or most users pass both headers to select a format? Could you please add a <Note> below to clarify?

Reply:
```suggestion
* Sending `Apollo-Query-Plan-Experimental-Format` with one of the supported options changes the output format:
```